scripts/onload.js

```diff
@@ -115,10 +115,11 @@ map.on('load', function() {
 			'polygonLayerName': 'texas-charter-companies-poly',
 			'polygonFillColor': 'rgba(124, 124, 124, 0)',
 			'polygonOutlineColor': 'rgba(103, 65, 30, 0)',
-			'visibleOnLoad': false,
+			'visibleOnLoad': true,
 			'usedInZoomControl': true
 		}
 	);
+*/
```

Comment:
We should probably just cut these lines altogether, but for now I've commented them out in case there's some problem I had missed with removing the layer.

Reply:
seems prudent